app/components/wallet/add/option-dialog/WalletCreateOptionDialog.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+import type { Node } from 'react';
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import { defineMessages, intlShape } from 'react-intl';
+
+import Dialog from '../../../widgets/Dialog';
+import DialogCloseButton from '../../../widgets/DialogCloseButton';
+import OptionBlock from '../../../widgets/options/OptionBlock';
+import type { $npm$ReactIntl$IntlFormat } from 'react-intl';
+import styles from '../../../widgets/options/OptionListWrapperStyle.scss';
+
+const messages = defineMessages({
+  dialogTitle: {
+    id: 'wallet.add.optionDialog.create.dialogTitle',
+    defaultMessage: '!!!Create',
+  },
+  createNormalTitle: {
+    id: 'wallet.add.optionDialog.create.normalWallet.title',
+    defaultMessage: '!!!Create a Wallet',
+  },
+  createNormalDescription: {
+    id: 'wallet.add.optionDialog.create.normalWallet.description',
+    defaultMessage: '!!!Choose this option to create your wallet.',
+  },
+  createPaperWalletTitle: {
+    id: 'wallet.add.optionDialog.create.paperWallet.title',
+    defaultMessage: '!!!Paper Wallet',
+  },
+  createPaperWalletDescription: {
+    id: 'wallet.add.optionDialog.create.paperWallet.description',
+    defaultMessage: '!!!If you want to generated a Yoroi paper wallet, you can choose this option to create the funds from your Yoroi paper wallet.',
```

Comment:
Not really sure about this wording. Maybe something like
```
Paper wallets can be created even on devices not connected to the internet which makes them well-suited for single-use cold storage.
```

However, note that we have a more in-depth explanation in `paper-wallets/intro/en-US.md`